packages/connect-core/src/entities/Organization.ts

```diff
@@ -37,13 +37,21 @@ export default class Organization {
     location: string,
     connector: ConnectorInterface,
     provider: any,
-    network: Networkish
+    network: Network | string
   ) {
     this.location = location
 
+    const getProvider = (): ethers.providers.Provider => {
```

Comment:
Should we name this `getEthersProvider()`? To make it clear that it is not an [Ethereum provider](https://eips.ethereum.org/EIPS/eip-1193) (which ethers calls a “Web3 Provider”, to make it more confusing :laughing:).